parsons/action_kit/action_kit.py

```diff
@@ -233,6 +233,59 @@ def update_event(self, event_id, **kwargs):
         )
         logger.info(f"{resp.status_code}: {event_id}")
 
+    def get_blackholed_email(self, email):
+        """
+        Get a blackholed email.
```

Comment:
Because "blackholing" an email is a bit of an ActionKit-specific term, it might be helpful to either have a brief description of what a blackholed email is or a link to the ActionKit docs for blackholed emails.

Definition per ActionKit:
> You may want to prevent certain email addresses or domains from receiving bulk and transactional messages from ActionKit.

Link: https://docs.actionkit.com/docs/manual/guide/mailings_tools.html#blackhole